proposals/2023-01-regen-ledger-v5.0-upgrade/README.md

```diff
@@ -0,0 +1,73 @@
+# Regen Ledger v5.0 Upgrade
+
+This is a software upgrade proposal for the upgrade to Regen Ledger `v5.0`. If passed, this proposal would commit Regen Mainnet to halting the application binary for Regen Ledger `v4.1` at approximately 16:00 UTC on January 17th at block height `8942600` and requiring nodes to restart with the application binary using Regen Ledger `v5.0`.
+
+Validators and node operators can refer to [Upgrade Guide v5.0](https://docs.regen.network/validators/upgrade/v5.0-upgrade.html) for guidelines on performing the upgrade.
```

Comment:
```suggestion
Validators and node operators can refer to [Upgrade Guide v5.0](https://docs.regen.network/validators/upgrades/v5.0-upgrade.html) for guidelines on performing the upgrade.
```

Was getting a 404, this link should work:
https://docs.regen.network/validators/upgrades/v5.0-upgrade.html